src/api/lib/tasks/watchlist_migration.rake

```diff
@@ -0,0 +1,33 @@
+namespace :watchlist do
+  desc 'Create a list of emails which should be used to warn users'
+  task list_users_to_warn: :environment do
+    users_emails = []
+    # Retrieve the emails from users which, having projects in the new watchlist, also
+    # have some project in the old watch list which aren't in the new watchlist.
+    User.where(id: WatchedItem.where(watchable_type: :project).map(&:user_id).uniq).each do |user|
+      watched_project_items = user.watched_items.where(watchable_type: :project).map(&:watchable)
+      next if watched_project_items.length.zero?
+
+      watched_projects = user.watched_projects.map(&:project)
+      extra = watched_projects - watched_project_items
+      next if extra.length.zero?
+
+      users_emails.push(user.email)
+    end
+    puts 'List of emails to send the warn of the watchlist migration: ' + users_emails.join(',')
```

Comment:
How are you are going to send this email? Manually?

Reply:
Yes.

Reply:
Why not do it from here? It'll be easier no?

Reply:
To do it from here could mean to [generate a Mailer](https://guides.rubyonrails.org/action_mailer_basics.html). What did you have in mind?

Reply:
We talked offline and will skip this.